addons-cluster/postgresql/templates/_helpers.tpl

```diff
@@ -18,7 +18,9 @@ Define postgresql ComponentSpec with ComponentDefinition.
   clusterDefinitionRef: postgresql
   topology: {{ .Values.mode }}
   componentSpecs:
-    - name: postgresql
+    - name: {{ include "postgresql-cluster.component-name" . }}
+      labels:
+        {{- include "postgresql-cluster.patroni-scope-label" . | indent 8 }}
```

Comment:
how to specify the pg version when install the chart?

Reply:
> how to specify the pg version when install the chart?

This is not within the scope of this PR. The version will be specified through serviceVersion in the future, but the management of serviceVersion requires further design.